### PR TITLE
Update from Bootstrap 3 to 5

### PR DESCRIPTION
**Pull request by sebastiancarlos**
Very similar to the update done to taskwarrior.org: https://github.com/GothenburgBitFactory/tw.org/pull/930

### Screenshots 
<details>
  <summary>Expand</summary>

main
docs_summary
reference_timew-start 1_

</details>

### Notes:
- Code blocks were kept with a dark background. This is because we need them to have contrast with white text, which appears sometimes when displaying chart examples.
- But code blocks were kept with a light background in the rendered man pages.

### CODE REVIEW

**Review by lauft — APPROVED** [review]
LGTM

File: themes/bootstrap/layouts/_default/releases.html
Comment (mattsmida) [inline themes/bootstrap/layouts/_default/releases.html:1]: Nice catch on the non-wrapping SHA256 hashes.

**Comment by mattsmida** [comment]
Thanks Sebastian! Great to see the polish here on the timewarrior site in addition to your work on the taskwarrior site.

I'm merging these changes now, and I'm going to add two small things that I noticed during my review in another PR:
1. tw.org does not include a bootstrap-theme CSS file like ti.net does, leading to some inconsistencies between the two sites. So I'm going to remove that CSS file for ti.net.
2. The footer for ti.net appears smaller than that of tw.org. I'm going to update the CSS to get them to match.

**Review by mattsmida — COMMENTED** [review]
Looks good.

**Pull request merged**